src/cookies.ts

```diff
@@ -17,21 +21,51 @@ const getDomainAttribute = (): string => {
     return shortDomain === 'localhost' ? '' : shortDomain;
 };
 
-const addCookie = (name: string, value: string, daysToLive?: number): void => {
+const addCookie = (name: string, value: string | GuPurposeState): void => {
     const options: {
         domain: string;
-        expires?: number;
+        expires: number;
     } = {
         domain: getDomainAttribute(),
+        expires: COOKIE_MAX_AGE,
     };
 
-    if (daysToLive) {
-        options.expires = daysToLive;
+    Cookies.set(name, value, options);
+};
+
+const getCookie = (name: string): string | undefined => Cookies.get(name);
+
+const readGuCookie = (): GuPurposeState | null => {
```

Comment:
How about using this https://www.npmjs.com/package/js-cookie#json ?

Reply:
Good suggestion!